angular-workspace/projects/ni/nimble-angular/table-column/anchor/nimble-table-column-anchor.directive.ts

```diff
@@ -1,9 +1,10 @@
 import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
 import type { TableColumnAnchor } from '@ni/nimble-components/dist/esm/table-column/anchor';
-import type { AnchorAppearance } from '../../anchor/nimble-anchor.directive';
-import { BooleanValueOrAttribute, NumberValueOrAttribute, toBooleanProperty, toNullableNumberProperty } from '../../utilities/template-value-helpers';
+import { AnchorAppearance } from '@ni/nimble-components/dist/esm/anchor/types';
```

Comment:
I'm trying to understand what's going on here. Previously it was `nimble-anchor.directive` that re-exported the `AnchorAppearance` type from `nimble-components`. And `nimble-table-column-anchor.directive` was able to import it from there. But now `nimble-table-column-anchor.directive` is using the type from `nimble-components` and also re-exporting it.

Is this change necessary because this can't refer to the type from the `nimble-anchor.directive` in the main entry point? Or it could, but we like this pattern better because it means that clients can import everything they need from this entry point rather than having to import the type from one entry point and the directive from another?

Reply:
I'm pretty sure the previous code was incorrect -- the anchor column should not have been depending on the anchor directive to export a type that is needed to use the column. When the type was needed by two directives (the anchor directive and the anchor column directive), the export of the type should have been moved to `public-api.ts` for the main entry point. We have [a section in that file](https://github.com/ni/nimble/blob/980eb3cc9d18d2c8c855f784180a6fe1925947a3/angular-workspace/projects/ni/nimble-angular/src/public-api.ts#L112) that exports types that are needed by multiple components.

To summarize & answer some of your specific questions -- even if we could refer to the type from `nimble-anchor.directive` from here (which we can't), that wouldn't be the correct approach. The pattern introduced here is preferred because clients an import everything they need to use a component from a single entry point.

Reply:
Makes sense, thanks!